misc/config_tools/schema/types.xsd

```diff
@@ -192,21 +221,35 @@ device in VM . Set in hex.</xs:documentation>
 </xs:simpleType>
 
 <xs:complexType name="IVSHMEMRegionType">
-  <xs:sequence>
+  <xs:all>
+    <xs:element name="NAME">
+      <xs:annotation acrn:title="Name">
+        <xs:documentation>Name of the shared memory region.</xs:documentation>
+      </xs:annotation>
+      <xs:simpleType>
+	<xs:restriction base="xs:string">
+	  <xs:pattern value="\w+" />
+	</xs:restriction>
+      </xs:simpleType>
+    </xs:element>
+    <xs:element name="PROVIDED_BY" type="ProviderType" default="Hypervisor">
+      <xs:annotation acrn:title="Emulated by">
+        <xs:documentation>Whether the shared memory region is provided by the hypervisor or device model.</xs:documentation>
```

Comment:
If the title says "emulated by", this description should user the same verb.  Pre-launched VMs can't use "device model" here, right?